joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/utils/packageTable.scala

```diff
@@ -0,0 +1,42 @@
+package io.joern.rubysrc2cpg.utils
+
+import scala.collection.mutable
+
+case class MethodTableModel(moduleName: String, methodName: String, parentClassPath: String) {
```

Comment:
The Data structures defined in this class `packageTable.scala` have redundant properties, can we optimize the schema?

Reply:
yes resolved after changing to hashMap.